src/components/map/ha-map.ts

```diff
@@ -469,26 +477,30 @@ export class HaMap extends ReactiveElement {
       }
       #map.dark {
         background: #090909;
-        --map-filter: invert(0.9) hue-rotate(170deg) grayscale(0.7);
+        color: #ffffff;
+        --map-filter: invert(0.9) hue-rotate(170deg) brightness(1.5)
+          contrast(1.2) saturate(0.3);
+      }
+      #map.light {
+        background: #ffffff;
+        color: #000000;
```

Comment:
I think we should only use hard coded colors when the map darkmode setting differs from the theme darkmode. Otherwise we should just use theme colors

Reply:
The `#000000` is needed to show the icon of the home (house) in black, when HA is in dark mode and map should use the light theme. I will change it to `--white-color`. (`#ffffff` → `--black-color`)